examples/angular-example/.vscode/tasks.json

```
@@ -0,0 +1,42 @@
+{
+  // For more information, visit: https://go.microsoft.com/fwlink/?LinkId=733558
```

Comments aren't allowed this way in JSON. Is it possible to use either one of these two solutions instead?
- [`JSON5`](https://json5.org/)
- using a hack to define this like this (does the functionality still work in VS Code?):
```suggestion
  "_for_more_information_visit": "https://go.microsoft.com/fwlink/?LinkId=733558"
```